docs/releasehistory.md

```diff
@@ -9,12 +9,49 @@ Releases follow versioning as described in
 
 Please note that all releases prior to a version 1.0.0 are considered pre-releases and many API changes will come before a stable release.
 
-## 0.2.0 - Development
+## 0.2.1 - Current development
 
-The v0.2.x line of Interchange targets biopolymer support and consistitutes several significant
-changes compared to older versions. Because of this, versions v0.2.0 and newer will not generally be compatible with the 0.1.x line or versions of the OpenFF Toolkit older than v0.11.0.
 
-The default branch has been renamed from `master` to `main`.
+## 0.2.0 - 2022-08-29
+
+The 0.2.x line of Interchange targets biopolymer support alongside version 0.11.0 of the OpenFF Toolkit.
+Due to the scope of changes, versions 0.2.0 and newer will not generally be compatible with the 0.1.x line or versions of the OpenFF Toolkit less than 0.11.0.
+In lieu of a changelog entry for this release, below is a brief summary of the current capabilities of Interchange.
```

Comment:
(optional) I might just be really tired, but I read over this quickly and missed this part, then I was confused about why the changelog listed a bunch of things that already exist. 

```suggestion
In lieu of a changelog entry for this release, **below is a brief summary of the current capabilities of Interchange.**
```